BrowserKit/Sources/SiteImageView/ImageProcessing/ImageError.swift

```diff
@@ -5,12 +5,29 @@
 import Foundation
 
 enum ImageError: Error, CustomStringConvertible {
+
+    enum BundleError: CustomStringConvertible {
+        var description: String {
+            switch self {
+            case .noBundleRetrieved(let error), .imageFormatting(let error), .noImage(let error):
+                return error
+            }
+        }
+
+        case noBundleRetrieved(String)
+        case imageFormatting(String)
+        case noImage(String)
+    }
+
     case unableToDownloadImage(String)
+    case unableToGetFromBundle(BundleError)
```

Comment:
You can just throw a BundleError directly, no need to tie it to ImageError like this.